packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.component.tsx

```diff
@@ -123,7 +140,27 @@ const ClickablePatientContainer = ({ patient, children }: ClickablePatientContai
     patientClickSideEffect?.(patientUuid, patient);
   }, [patientClickSideEffect, patientUuid, patient]);
 
-  if (nonNavigationSelectPatientAction) {
+  if (context2) {
+    return (
+      <button
+        className={classNames(styles.patientBannerButton, styles.patientBanner, {
+          [styles.patientAvatarButton]: nonNavigationSelectPatientAction,
+        })}
+        key={patientUuid}
+        onClick={() => {
+          if (onPatientSelected) {
+            onPatientSelected(patient.id, patient, launchChildWorkspace, closeWorkspace);
+          } else {
+            navigate({
+              to: config.search.patientChartUrl,
+              templateParams: { patientUuid },
+            });
+          }
```

Comment:
API-wise and maybe longer-term, it might be nice if `onPatientSelected` returned a boolean indicating whether or not to also navigate to the chart. That way it would be possible to use the callback to do other things, e.g., pre-populate the patient context, data-fetching, or some other action, but still navigate to the chart.